.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
```

Comment:
Given that our primary audience are QE, I would expect most of readers to intuitively differentiate between "problem" and "feature request" (because bug tracking systems tirelessly taught us there is a difference).
Maybe

> Please describe your feature/problem/idea